lang/en/block_opencast.php

```diff
@@ -263,17 +283,26 @@
 $string['importvideos_importseriescleanupfailed'] = 'At least one of the existing Opencast series modules within this course could not be cleaned up. Please try again or contact your Moodle administrator.';
 $string['importvideos_processingexplanation'] = 'These video files will be duplicated in Opencast and then made available in this course.';
 $string['importvideos_sectionexplanation'] = 'In this section, you can import existing video files from other Moodle courses to this Moodle course.';
+$string['importvideos_aclprocessingexplanation'] = 'The seriesid of the selected course will be used in this course as secondary seriesid and the ACLs of its videos will be changed accordingly.';
+$string['importvideos_aclsectionexplanation'] = 'In this section, you can import existing video files from other Moodle courses to this Moodle course.';
 $string['importvideos_settingcoreenabled'] = 'Allow video import within Moodle core course import wizard';
-$string['importvideos_settingcoreenabled_desc'] = 'If enabled, teachers can import existing video files from other Moodle courses to their Moodle course by using the Moodle core course import wizard. Within the wizard, there is an additional option shown to import the videos which have been uploaded within the Opencast block in the other course. Using this feature, the teacher can import all videos from another course as bulk, but he cannot select individual videos.<br />The videos are duplicated in Opencast by an ad-hoc task in Moodle and will show up in the course\'s video list with a short delay.';
+$string['importvideos_settingcoreenabled_desc'] = 'If enabled, teachers can import existing video files from other Moodle courses to their Moodle course by using the Moodle core course import wizard. Within the wizard, there is an additional option shown to import the videos which have been uploaded within the Opencast block in the other course. Using this feature, the teacher can import all videos from another course as bulk, but he cannot select individual videos.<br />When Dupliaction Events is selected as the Import mode then the videos are duplicated in Opencast by an ad-hoc task in Moodle and will show up in the course\'s video list with a short delay.<br />In case of ACL Change as Import mode, the series id of the course being imported is used but the ACL of the video will be changed in order to grant access within the new course.';
 $string['importvideos_settingenabled'] = 'Allow video import';
 $string['importvideos_settingenabled_desc'] = 'If enabled, teachers can import existing video files from other Moodle courses to their Moodle course. You have to further set a duplication workflow and enable one or both of the two following settings to manually import videos or to import videos within the Moodle core course import wizard. If you don\'t enable any of these, this setting will have no effect.';
```

Comment:
Thats still tailored to the duplication mode although it is used for both. Please use a more generic text for 'importvideos_settingenabled_desc'.